src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.html

```diff
@@ -102,12 +102,11 @@
             <div class="kuiVerticalRhythmSmall">
               <select
                 class="kuiSelect kuiSelect--large kuiVerticalRhythmSmall"
-                data-test-subj="createIndexPatternTimeFieldSelect || !controller.indexHasDateFields"
```

Comment:
`data-test-subject` can contain multiple space-separated properties, so you could have groups and such :)